src/CMakeLists.txt

```diff
@@ -232,8 +232,11 @@ IF (USE_HTTPS)
 		SET(GIT_MBEDTLS 1)
 		LIST(APPEND LIBGIT2_INCLUDES ${MBEDTLS_INCLUDE_DIR})
 		LIST(APPEND LIBGIT2_LIBS ${MBEDTLS_LIBRARIES})
-		LIST(APPEND LIBGIT2_PC_LIBS ${MBEDTLS_LDFLAGS})
-		LIST(APPEND LIBGIT2_PC_REQUIRES "mbedtls")
+		# mbedTLS has no pkgconfig file, hence we can't require it
+		# https://github.com/ARMmbed/mbedtls/issues/228
+		# For now, pass its link flags as our own
+		LIST(APPEND LIBGIT2_PC_LIBS ${MBEDTLS_LIBRARIES})
```

Comment:
Just to be sure here: does `MBEDTLS_LIBRARIES` include `MBEDTLS_LDFLAGS`?

Reply:
It's a little bit weirder : the FindmbedTLS module doesn't actually set `MBEDTLS_LDFLAGS` [at all](https://github.com/libgit2/libgit2/blob/master/cmake/Modules/FindmbedTLS.cmake#L11), so I hope this is sufficient. AFAIU, `Libs.private` only impacts static linking (ie. pkg-config libgit2 --static --libs`), and the linker should already know what to do when dyn-linking.

The resulting pc file looks like this :
```
prefix="/usr/local"
libdir=${prefix}/lib
includedir=${prefix}/include

Name: libgit2
Description: The git library, take 2
Version: 0.27.0

Libs: -L${libdir} -lgit2
Libs.private: -lcurl -L/usr/local/lib -lmbedtls -lmbedx509 -lmbedcrypto -lz -L/usr/local/Cellar/libssh2/1.8.0/lib -lssh2 -L/usr/lib -liconv
Requires.private: 

Cflags: -I${includedir}
```